scss/bootstrap/overrides/style/_buttons.scss

```diff
@@ -90,12 +81,14 @@
 }
 
 .btn-danger {
-
   &:hover {
     background-color: darken($btn-danger-bg, $crm-darken-percentage);
   }
 }
 
+.btn .caret {
```

Comment:
what is this `.caret` class coming from? Is it from bootstrap?

Reply:
Yes: https://getbootstrap.com/docs/3.3/components/#dropdowns